pwiz_tools/Skyline/SkylineTester/CreateZipInstallerWindow.cs

```diff
@@ -186,8 +186,12 @@ public static void CreateZipFile(string zipPath)
                             AddFile(file, zipFile);
                     }
 
+                    // MCC 2/14/2023: commented out adding test zips after discussion with Brendan that if we need
+                    // to test Skyline outside a source tree, we can find a way to get the zip files on the fly or
+                    // have a separate artifact for that. This will cut SkylineTester.zip from ~600MB to ~100MB.
+
                     // Add test zip files.
-                    var zipFilesList = new List<string>();
+                    /*var zipFilesList = new List<string>();

```

Comment:
Instead, please add an argument to the function for whether or not to include the test files, and have the command-line pass in false. That will make it simpler to connect functionality that re-enables this feature, either by adding a command-line argument to make the value true, or by connecting this form and adding a checkbox for whether or not to include test data.

Reply:
OK. Do you want me to implement the UI, or just add the function param?